modules/electrophysiology_browser/README.md

```diff
@@ -16,20 +17,24 @@ been correctly inserted into LORIS.
 
 The Electrophysiology Browser displays electrophysiology datasets that have been
 inserted into LORIS from a BIDS-format collection. Derived or processed electrophysiology
-datasets can also be accessed via this module.
+datasets can also be accessed and edited via this module.
 
 ## Permissions
 
-The Electrophysiology Browser uses the following permissions. Any of them is 
-sufficient to provide access to view data in the module.
+The Electrophysiology Browser uses the following permissions. Either of the first two is 
+sufficient to provide access to view data in the module. The third permission provides editing
+permissions for derivative datasets.
```

Comment:
```suggestion
permissions to add or modify annotations (derivatives) for data from all sites in this module.
```
@AlexandraLivadas  can you update these words `from all sites` if not accurate ?  does your permission to add/edit on all-sites or just the user's affiliated sites?

Reply:
I will update this because it does not apply for all-sites. The 2 permissions that existed before determine if the session can be viewed and this permission just determines if the annotations of the session being viewed can be edited.